### DISCUSSION TITLE
How do I query entries, even if they are disabled in GraphQL?

### DISCUSSION THREAD

**Discussion by toddpadwick** [craftcms/cms#9953] [Q&A] [answered]
I have a 'services' section on our website. Some entries which we have yet to put together enough content, do not have pages so we have disabled them. But we still want to render the service titles and descriptions under each of our case studies which have them assigned via an entries field. The intention would be that those entries that are not enabled will still render, but won't have an anchor link.

How might I achieve this with GraphQL? I cannot seem to figure out how to query entries that are disabled in GraphQL. 
What I need is:
- to be able to query all entries even if disabled
- to be able to determine which entries are disabled and which are not in the query response

**Comment by andris-sevcenko** [comment, marked as answer]
If the schema has the "Allow listing non-live and otherwise inactive elements" setting enabled, you should be able to do
```graphql
{
  entries (status:["live","disabled"]) {
    id
    title
    status
  }
}
```

**Reply by toddpadwick** [reply]
Ah fantastic. Thank you!
